webapp/components/survey/NodeDefDetails/TaxonProps.js

```diff
@@ -98,7 +98,7 @@ const TaxonProps = (props) => {
             }}
             header={i18n.t('taxonomy.header')}
           >
-            <TaxonomyView showClose={false} />
+            <TaxonomyComponent showClose={false} />
```

Comment:
I put TaxonomyComponnet due to we are using the Taxonomy from Core.

Reply:
maybe call this TaxonomyDetails . more consistent with NodeDefDetails as well